index.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('#hamburger');
+const closetog = document.querySelector('#togclose');
+const toggleMenu = document.querySelector('.toggle-menu');
+
+function show() {
+  toggleMenu.style.display = 'flex';
+  toggleMenu.style.top = '0';
+}
+
+function close() {
+  toggleMenu.style.top = '-100%';
+}
+
+hamburger.addEventListener('click', show);
+closetog.addEventListener('click', close);
```

Comment:
- [ ] Good job 👏  You implemented all features in the requirements except only one. Kindly add a feature that `when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears`.

Reply:
Feature implemented